providers/deprecationProvider.js

```
@@ -0,0 +1,19 @@
+module.exports = function (context, execution, controller) {
+  if (!context.module && !context.modules) {
+    Object.defineProperty(context, 'module', {
+      get: function () {
+        console.warn('Cerebral 0.34 uses a provider to access "module" and it has become more powerful. Please install: https://github.com/cerebral/cerebral-provider-modules')
+        return null
```

maybe better just throw error here instead of return?